test.sh

```diff
@@ -1,6 +1,7 @@
 set -e
 
 bun install --frozen-lockfile
+bun tsc --noEmit
 bun biome ci
```

Comment:
```suggestion
bun biome ci
bun tsc
```
I think biome should go first since it's so fast, and secondly we shouldn't need `--noEmit` as we already have that set in `tsconfig.json`

Reply:
oh missed that, thanks